spec/mock-request.js

```diff
@@ -1,14 +1,26 @@
 "use strict";
 const q = require("q");
+let clock = jasmine.Clock;
```

Comment:
Why are you aliasing `jasmine.Clock`? It's useful to know at the callsites that you're manipulating jasmine's stuff and not our own, and the alias removes this context.